rundeck-data-model/src/main/java/org/rundeck/app/data/model/v1/webhook/dto/SaveWebhookResponse.java

```diff
@@ -0,0 +1,15 @@
+package org.rundeck.app.data.model.v1.webhook.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.rundeck.app.data.model.v1.webhook.RdWebhook;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class SaveWebhookResponse {
+    RdWebhook webhook;
+    boolean isSaved;
+    Object errors;
```

Comment:
Be more specific on the type here. A DTO should not declare a field as a generic Object.

Reply:
Changed to type String to hold the error messages